e2e_postgres_test.go

```diff
@@ -738,8 +738,8 @@ func TestPostgresAuthentication(t *testing.T) {
 	var opts []cloudsqlconn.Option
 	if ipType != "private" {
 		creds = keyfile(t)
-		opts = AddIPTypeOptions(opts)
 	}
+	opts = AddIPTypeOptions(opts)
```

Comment:
nit: this should be called `addIpTypeOptions`, in Go you only capitalize the first letter of a func if you want it to be exported, I am pretty sure this func is only used within this file (i.e. is internal)?

```suggestion
	opts = addIpTypeOptions(opts)
```

Reply:
Noted, thanks!